src/styles/index.ts

```diff
@@ -4914,6 +4930,19 @@ const styles = (theme: ThemeColors) =>
             flex: 1,
         },
 
+        tripReservationIconContainer: {
+            width: 32,
+            height: 32,
+            backgroundColor: theme.overlay,
```

Comment:
I don't think this is the correct value. We want to use the same color we use for borders.

Reply:
In figma, it's border color:

<img width="1079" alt="image" src="https://github.com/Expensify/App/assets/29673073/28be0476-6192-4b40-88ac-01c3f53a6b58">

Reply:
Correct, it shouldn't be overlay.